iotdb/src/main/java/org/apache/iotdb/db/engine/memtable/MemTableFlushTaskV2.java

```diff
@@ -94,14 +96,18 @@ public void flushMemTable() {
         "Storage group {} memtable {}, flushing into disk: data sort time cost {} ms.",
         storageGroup, memTable.getVersion(), sortTime);
 
+    Boolean success = false;
     try {
-      ioFlushTaskFuture.get();
+      success = ioFlushTaskFuture.get();
```

Comment:
Rename this parameter to `flushSuceeded`

Reply:
renamed as `flushSuccessfully`